### PR TITLE
More STYLE.md updates

### PR DESCRIPTION
**Pull request by jdchristensen**
Typos I noticed today, plus a couple of obsolete things.

I pretty sure that the whole last part of the bug minimizer discussion is not needed anymore, since it was about using hoqc, which we no longer use.  @JasonGross , let me know if I'm right about that.  (The third commit below.)

### CODE REVIEW

**Comment by Alizter** [comment]
Changes LGTM. Would you also be able to hard-wrap some of the paragraphs? I think we wrap at 80 (emacs default).

**Comment by jdchristensen** [comment]
Lines wrapped.  Should we wait to hear what Jason says about the bug minimizer, or just go ahead and merge?

File: STYLE.md [hunk 1]
```diff
@@ -1428,13 +1444,12 @@ You will need to pass the bug-finder several arguments to tell it to
 use the HoTT version of Coq and where to find the rest of the library;
```
Comment (JasonGross) [inline STYLE.md:1444]:
```suggestion
pass the right flags and where to find the rest of the library;
```

File: STYLE.md [hunk 2]
```diff
@@ -1428,13 +1444,12 @@ You will need to pass the bug-finder several arguments to tell it to
...
 a common invocation would be something like
 
-    $ /path/to/find-bug.py --coqc ../hoqc --coqtop ../hoqtop -R . HoTT Path/To/Buggy.v bug_minimized.v
+    $ /path/to/find-bug.py --coqc .../coqc --coqtop .../coqtop -R . HoTT Path/To/Buggy.v bug_minimized.v
```
Comment (JasonGross) [inline STYLE.md:1447]:
```suggestion
    $ /path/to/find-bug.py --arg -noinit --arg -indices-matter -R . HoTT Path/To/Buggy.v bug_minimized.v
```

Comment (Alizter) [reply STYLE.md:1447]: Alternatively you can use `-f`(?) to give `_CoqProject` instead.

**Comment by jdchristensen** [comment]
I'll go ahead and merge unless I hear any other suggestions.

**Pull request merged**